insights/parsers/libssh_config.py

```diff
@@ -0,0 +1,128 @@
+"""
+Libssh configuration files
+==========================
+
+This module provides the ssh options in libssh configuration file
+
+Parsers provided by this module are:
+
+LibsshClientConfig - file ``/etc/libssh/libssh_client.config``
+--------------------------------------------------------------
+
+LibsshServerConfig - file ``/etc/libssh/libssh_server.config``
+--------------------------------------------------------------
+"""
+
+from insights.specs import Specs
+from insights.core import CommandParser
+from .. import parser, get_active_lines
+from insights.parsers import SkipException
+
+
+class LibsshConfig(CommandParser):
+    """
+    Base class for libssh configuration file.
+
+    There are many options in the libssh configuration file and
+    all of them are stored in one of the following formations:
+        a)   Key  [space]  Value
+        b)   Key  =  Value
+
+    There are the keywords used in the configuration file:
```

Comment:
@shlao you have CI errors that I believe are caused by lines 27 and 31 missing `::` and a blank line after.